packages/api/endpoints/granules.js

```diff
@@ -928,7 +865,6 @@ async function bulkOperations(req, res) {
       payload,
       type: 'BULK_GRANULE',
```

Comment:
Please manually test bulk operations from dashboard. 1. Select a list of granules 2. Use query from Metrics, you can create granules in your database with the granuleIds match the ones from [metrics query.](https://nasa.github.io/cumulus/docs/next/operator-docs/bulk-operations/#using-a-kibana-query)